pkgs/by-name/fr/frr/package.nix

```diff
@@ -81,9 +81,6 @@
 , ospfApi ? true
 }:
 
-lib.warnIf (!(stdenv.buildPlatform.canExecute stdenv.hostPlatform))
```

Comment:
If explicitly enabling `snmpSupport`, does the build fail or is the feature silently disabled?

Reply:
Fails without finding `net-snmp-config`, I now just fixed snmpSupport on cross  [`9f2266d` (#362546)](https://github.com/NixOS/nixpkgs/pull/362546/commits/9f2266dba97a2df361614788c8cd76738386e0f5)

Reply:
That's even better! Just wanted to make sure no one's expectations are silently subverted…